_inc/client/main.jsx

```diff
@@ -93,7 +93,8 @@ class Main extends React.Component {
 		if ( tracksUser ) {
 			analytics.initialize(
 				tracksUser.userid,
-				tracksUser.username
+				tracksUser.username,
+        tracksUser.blogid,
```

Comment:
There seem to be some indentation issues here - this needs tabs instead of spaces.